sdk/spring/spring-cloud-azure-supported-spring.json

```diff
@@ -5,13 +5,21 @@
     "snapshot" : true,
     "supportStatus" : "TODO",
     "spring-boot-version" : "3.0.0-SNAPSHOT",
-    "spring-cloud-version" : "2022.0.0-M5"
+    "spring-cloud-version" : "2022.0.0-RC2"
   },
   {
     "current" : true,
     "releaseStatus" : "PRERELEASE",
     "snapshot" : false,
     "supportStatus" : "SUPPORTED",
+    "spring-boot-version" : "3.0.0-RC2",
```

Comment:
can we add some metadata information in the json file?
like where and how this json file is used

Reply:
Used from 
https://github.com/Azure/azure-sdk-for-java/blob/cf314433c24072b189d8fd2a2f608e9a5481e2f7/sdk/spring/scripts/compatibility_update_supported_version_matrix_json.py#L49, 
It's better to add more doc descriptions in yml or py files, @Netyyyy  please help add usage docs, thanks!